frontend/uddhar-frontend/src/coordinator/components/TeamDetails.jsx

```
@@ -0,0 +1,91 @@
+  locations,
+  responsibilities,
+  handleSelect,
+  assign,
+}) => {
+  return (
+    <div className="fixed inset-0 bg-blue bg-opacity-50 flex items-center justify-center p-4 z-50">
+      <div className="bg-white rounded-lg p-6 w-full max-w-2xl relative">
+        <button
+          onClick={closeTeamDetails}
+          className="absolute top-4 right-4 text-gray-500 hover:text-gray-700 text-2xl cursor-pointer"
+        >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)